### PR TITLE
Remove reader/post-options

### PR DESCRIPTION
**Pull request by bluefuton**
The component `reader/post-options` originally provided the ellipsis menu on Reader post cards (containing functions like 'Follow' and 'Edit').

The component was migrated to `blocks/reader-post-options-menu` during the Reader refresh, and the old version in `reader/post-options` is now redundant.

### To test

On a Reader post card, click the ellipsis menu and ensure the menu functions as expected.

screen shot 2017-01-05 at 16 02 26

### CODE REVIEW

**Comment by blowery** [comment]
Taking a peek at this.

**Comment by blowery** [comment]
Alignment looks a little wonky on calypso.live:

following_ _reader_ _wordpress_com

File: client/reader/post-options/_style.scss [hunk 1]
```diff
@@ -1,91 +0,0 @@
...
-		transition: all 0.15s cubic-bezier(0.175, 0.885, 0.32, 1.275);
-	}
-
-	&:hover .gridicon,
-	&:focus .gridicon {
-		fill: $white;
-	}
-}
-
-.popover__menu .follow-button {
```
Comment (blowery) [inline client/reader/post-options/_style.scss]: Hrm. This was unscoped, so removing it might have more of an effect than we intend..

Comment (bluefuton) [inline client/reader/post-options/_style.scss]: Good spot! I'll move those styles into the new block.

**Comment by bluefuton** [comment]
@blowery I've ported across those styles from the old component and scoped them. Ready for 👀  again!

File: client/blocks/reader-post-options-menu/style.scss [hunk 2]
```diff
@@ -21,18 +21,41 @@
 	background: lighten( $gray, 30% );
 }
 
-.reader-post-options-menu__ellipsis-menu .follow-button {
+.reader-post-options-menu__popover {
 
-	&.is-following {
+	.popover__menu .follow-button {
```
Comment (blowery) [inline client/blocks/reader-post-options-menu/style.scss]: yay scoped!

**Pull request merged**